### PR TITLE
Add helm chart to deploy on kubernetes

### PR DESCRIPTION
**Pull request by smeruelo**

### CODE REVIEW

**Comment by smeruelo** [comment]
@dpordomingo @rporres PTAL

File: helm-charts/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+* Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the chart
+
+Some parameters must be provided.
+
+```
+helm install -n <release-name> <chart path> --set \
+ingress.hosts={spark-api.sourced.tech,go-siva.sourced.tech,enry.sourced.tech,bblf.sh},\
```
Comment (dpordomingo) [inline helm-charts/docs/README.md]: /spark-api/engine/s
/go-siva/siva/s

**Review by dpordomingo — APPROVED** [review]
I'd update the `README.md` with the new domains

**Review by rporres — APPROVED** [review]

**Comment by smeruelo** [comment]
Readme updated.

**Pull request merged**